src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj

```diff
@@ -11,6 +11,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <!-- Set this to false because assemblies should not reference this assembly directly, (except for tests, of course). -->
     <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
+    <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
```

Comment:
Maybe add a comment here for clarity/posterity?

Reply:
Good call!